### ISSUE TITLE
Evaluate Open Compute Project

### ISSUE THREAD

**Issue by joachimweyl** [OCP-on-NERC/operations#145]
Flax can provide us Open Compute Hardware. This is lower-priced hardware that fits the [Open Compute Project](https://www.opencompute.org/) conventions.
There is a rack of Open Compute hardware from Flax available for us to "play" with in the sandbox.
What steps are needed to enable the ops team to feel comfortable using this hardware?
Perhaps stand up an OpenShift and/or OpenStack cluster?

**Comment by joachimweyl** [comment]
@jtriley love to hear your thoughts.

**Comment by larsks** [comment]
@joachimweyl is this test hardware still available? If not we should close this issue.

**Comment by joachimweyl** [comment]
@er1p is this hardware still available?

**Comment by hpdempsey** [comment]
I believe it just became available, but we have no one available to work on
this now, since it is the start of the new semester and other projects are
more pressing.

**Comment by joachimweyl** [comment]
@er1p is this hardware still setup?

**Comment by joachimweyl** [comment]
nodes are still there. They will stay there until we have time to kick the tires. currently 2 of them are in use by another team but as soon as we have time to test them we should reach out to Flax and let them know when we can start. It will likely take them a few days to get back all the nodes for testing.

**Comment by hpdempsey** [comment]
Yes, this is definitely still in icebox, verging on never happening....
